### ISSUE TITLE
Cannot read property 'baseURL' of undefined

### ISSUE THREAD

**Issue by jonrouleau** [mikz/jspm-dev-server#8]
```
/[redacted]/node_modules/jspm-dev-server/bin/jspm-dev-server:31
options.baseURL = jspmConfig.loaderBrowser.baseURL
                                          ^

TypeError: Cannot read property 'baseURL' of undefined
    at Object.<anonymous> (/[redacted]/node_modules/jspm-dev-server/bin/jspm-dev-server:31:43)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Function.Module.runMain (module.js:575:10)
    at startup (node.js:160:18)
    at node.js:449:3
```

See https://github.com/jspm/jspm-cli/commit/ade725ea543befcd65bb2b91049d7a8bcc7933c7#diff-9db974009753841f4d6d2954b3cb77dcR145

**Comment by mikz** [comment]
Yep. Found the same issue and I was not sure if it is just me. Sorry.  Will take a look tomorrow.

**Comment by mikz** [comment]
Released as 0.4.1.
